VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
```

Comment:
>Please help me understand your point. To which changes are you referring? Please elaborate, perhaps including an example that helps clarify your objection.

There is insufficient information for some of the proposed additional types for them to be unambiguously converted to the equivalent set of BND records. Ideally, for every additional SVTYPE (and subtype if applicable), the specification would include an example variant call, together with the unique BND representation of that event. Critically, the proposed change allows any value in the SVTYPE field.

Consider a tandem duplication with 10 bp of novel inserted sequence between the duplications and the removal of 20bp of the duplicated sequence. According to the proposed changes, the following would be a valid VCF record representing the event:

```chr1       1000   complex_event     C       <what_goes_here_is_also_not_defined_because_the_PR_only_includes_changes_to_SVTYPE_and_not_ALT>   .       . SVTYPE=OTHER;END=3000```

The current specifications prohibit such ambiguous representations whereas the proposed changes allow such events.

Aside: in a similar manner, the specs should be updated to make fields such as END explicitly required if SVTYPE != BND. It currently only states that it is a reserved field, but doesn't actually require it to be written.